CONTRIBUTING.md

```diff
@@ -35,6 +35,7 @@ To send us a pull request, please:
 4. Commit to your fork using clear commit messages.
 5. Send us a pull request, answering any default questions in the pull request interface.
 6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
+7. Before merging, clean up the commit history for the PR. Each commit should be self-contained with an informative message, since each commit will be added to the istory for this project.
```

Comment:
```suggestion
7. Before merging, clean up the commit history for the PR. Each commit should be self-contained with an informative message, since each commit will be added to the history for this project.
```